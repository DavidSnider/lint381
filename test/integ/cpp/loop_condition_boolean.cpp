while (0);

while (1) {

}

while (foo) {}
