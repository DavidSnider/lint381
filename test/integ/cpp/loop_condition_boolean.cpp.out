loop_condition_boolean.cpp:1:7: error: Use 'false' instead of '0' in loop condition
while (0);
      ^^^
loop_condition_boolean.cpp:3:7: error: Use 'true' instead of '1' in loop condition
while (1) {
      ^^^
